typescript/agentkit/src/action-providers/twitter/twitterActionProvider.ts

```diff
@@ -69,12 +69,34 @@ export class TwitterActionProvider extends ActionProvider {
       throw new Error("TWITTER_ACCESS_TOKEN_SECRET is not configured.");
     }
 
-    this.client = new TwitterApi({
-      appKey: config.apiKey,
-      appSecret: config.apiSecret,
-      accessToken: config.accessToken,
-      accessSecret: config.accessTokenSecret,
-    } as TwitterApiTokens);
+    this.initializeClient(config);
```

Comment:
How does this fix the issue if it's just moving the instantiation into a function? `new TwitterApi` would still be called when the constructor runs, right?